### ISSUE TITLE
WOMtf happened to CWL in womtool?

### ISSUE THREAD

**Issue by aednichols** [broadinstitute/cromwell#4119]
```
java -jar /Users/anichols/Documents/Repos/cromwell/womtool/target/scala-2.12/womtool-35-fd560e9-SNAP.jar womgraph three_step.cwl
Failed to create wom Graph (reason 1 of 1): No getWomBundle method implemented in CWL v1
```

cf. [docs](https://cromwell.readthedocs.io/en/stable/WOMtool/), section Command Line Usage

**Comment by aednichols** [comment]
The `CwlV1_0LanguageFactory` can `validateNamespace` (used in `MaterializeWorkflowDescriptorActor`) but lacks any concept of `WomBundle`, the output of `getWomBundle` and the input to `createExecutable`

**Comment by dheiman** [comment]
If this ever does get fixed - the last version that didn't throw `No getWomBundle method implemented in CWL v1` (`31.1`) threw an error for me when I ran on https://github.com/NCI-GDC/gdc-dnaseq-cwl/blob/master/workflows/dnaseq/transform.cwl:

```
$ java -jar ~/bin/womtool-31.1.jar womgraph transform.cwl
Exception in thread "main" scala.MatchError: WomMaybePopulatedFileType (of class wom.types.WomMaybePopulatedFileType$)
	at womtool.graph.WomGraph$.fakeInput(WomGraph.scala:222)
	at womtool.graph.WomGraph$.$anonfun$womExecutableFromCwl$2(WomGraph.scala:205)
	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:234)
	at scala.collection.immutable.HashMap$HashMap1.foreach(HashMap.scala:231)
	at scala.collection.immutable.HashMap$HashTrieMap.foreach(HashMap.scala:462)
	at scala.collection.TraversableLike.map(TraversableLike.scala:234)
	at scala.collection.TraversableLike.map$(TraversableLike.scala:227)
	at scala.collection.AbstractTraversable.map(Traversable.scala:104)
	at womtool.graph.WomGraph$.$anonfun$womExecutableFromCwl$1(WomGraph.scala:205)
	at scala.util.Either.map(Either.scala:350)
	at womtool.graph.WomGraph$.womExecutableFromCwl(WomGraph.scala:201)
	at womtool.graph.WomGraph$.fromFiles(WomGraph.scala:172)
	at womtool.Main$.$anonfun$womGraph$2(Main.scala:98)
	at womtool.Main$.continueIf(Main.scala:102)
	at womtool.Main$.womGraph(Main.scala:96)
	at womtool.Main$.dispatchCommand(Main.scala:38)
	at womtool.Main$.delayedEndpoint$womtool$Main$1(Main.scala:167)
	at womtool.Main$delayedInit$body.apply(Main.scala:12)
	at scala.Function0.apply$mcV$sp(Function0.scala:34)
	at scala.Function0.apply$mcV$sp$(Function0.scala:34)
	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
	at scala.App.$anonfun$main$1$adapted(App.scala:76)
	at scala.collection.immutable.List.foreach(List.scala:389)
	at scala.App.main(App.scala:76)
	at scala.App.main$(App.scala:74)
	at womtool.Main$.main(Main.scala:12)
	at womtool.Main.main(Main.scala)
```

It would also be nice if the documentation included the fact that [`cwltool`](https://github.com/common-workflow-language/cwltool) needs to be installed.
